test/eredis_tcp_SUITE.erl

```diff
@@ -718,6 +752,22 @@ c() ->
     {ok, C} = Res,
     C.
 
+c_sentinel() ->
+    Res = eredis:start_link([{host, "127.0.0.8"},
+                             {port, ?WRONG_PORT},
+                             {database, 0},
+                             {password, ""},
+                             {reconnect_sleep, 100},
```

Comment:
When sentinel is used, typically host and port should not be specified, right? I think it's better to remove them here to make it simpler and more realistic. (But let's keep `?WRONG_PORT` in at least one test case like `t_connect_eredis_sentinel` to make sure it is covered too.)

Empty password, db 0, etc. are also redundant and tested elsewhere. The function `c/0` above uses default opts for everything.